### PR TITLE
Adding IMU Extrinsics

### PR DESCRIPTION
**Pull request by Serafadam**

### CODE REVIEW

**Comment by themarpe** [comment]
@MaticTonin @priamb 

Please review the above, thanks!

File: boards/OAK-D-LITE.json [hunk 1]
```diff
@@ -49,6 +49,43 @@
...
+        {   
+            "sensors":{ 
+                "BNO": {
+                    "name" : "BNO086",
+                    "extrinsics": {
+                        "to_cam": "CAM_B",
+                        "specTranslation": {
+                            "x": 0.0541,
+                            "y": 0.2,
+                            "z": -0.368
```
Comment (priamb) [inline boards/OAK-D-LITE.json:63]: Above extrinsic for BNO are not correct

File: boards/OAK-D-LITE.json [hunk 2]
```diff
@@ -49,6 +49,43 @@
...
+                    }
+                },
+                "BMI": {
+                    "name" : "BMI270",
+                    "extrinsics": {
+                        "to_cam": "CAM_B",
+                        "specTranslation": {
+                            "x": 0.6,
+                            "y": -0.2,
+                            "z": -0.6439
```
Comment (priamb) [inline boards/OAK-D-LITE.json:79]: Above extrinsic for BMI are not correct

File: boards/OAK-D-LR.json [hunk 3]
```diff
@@ -50,6 +50,43 @@
...
+        {   
+            "sensors":{ 
+                "BNO": {
+                    "name" : "BNO084",
+                    "extrinsics": {
+                        "to_cam": "CAM_B",
+                        "specTranslation": {
+                            "x": 12.1465,
+                            "y": 0.0,
+                            "z": -1.2616
```
Comment (priamb) [inline boards/OAK-D-LR.json:64]: Above extrinsic for BNO are not correct

**Review by MaticTonin — APPROVED** [review]
LGTM, all coordinates should be written in camera system

**Pull request merged**